terraform/applications/discourse/cdn.tf

```diff
@@ -12,6 +12,19 @@ resource "aws_cloudfront_distribution" "discourse" {
 
   origin {
     domain_name = local.workspace.discourse_cdn_zone
+    origin_id   = "discourse-pull-origin-old"
+    origin_path = ""
+
+    custom_origin_config {
+      http_port              = "80"
+      https_port             = "443"
+      origin_protocol_policy = "https-only" # Only talk to the origin over HTTPS
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
```

Comment:
We can probably trim this down to only include `TLSv1.2`. The origin here should prefer that and doesn't seem to support TLSv1 or TLSv1.1. https://www.ssllabs.com/ssltest/analyze.html?d=discourse.dev.mozit.cloud&s=52.26.54.173